src/main/features/dashboard/views/macro/claimStatus/helpWithFees.njk

```diff
@@ -141,21 +141,21 @@
   {% endset %}
   <p>{{ t('You will need your claim reference number: {{ ccdId }}', { ccdId: hyphenatedCcdCaseId }) }}</p>
 
-  {% if claim.claimData.interest.lastInterestCalculationDate %}
+  {# {% if claim.claimData.interest.lastInterestCalculationDate %}
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.